### ISSUE TITLE
Not Rendering Oxygen Gear or Parachute or view of world while rocket is in air

### ISSUE THREAD

**Issue by Dynamic225** [micdoodle8/Galacticraft#3484]
Minecraft version: 1.7.10
Galacticraft version: 3.0.12.504
Forge version: 10.13.4.1614

Playing in singleplayer

Other mods installed: Optifine, PlayerAPI, fastcraft, iChunUtil, MorePlayerModels (these are the ones I think are relevant; full list here: https://pastebin.com/wSahRPK4)

![2018-07-26_13 55 07](https://user-images.githubusercontent.com/34873036/43282605-32070c0e-90dc-11e8-8191-883952a59236.png)
Not rendering view of world while blasting off

![2018-07-26_13 55 51](https://user-images.githubusercontent.com/34873036/43282631-4a5c6858-90dc-11e8-8425-55e17d464c61.png)
![2018-07-26_13 58 27](https://user-images.githubusercontent.com/34873036/43282679-68b83110-90dc-11e8-8daa-388e980fd4a1.png)
same for moon while landing and taking off

![2018-07-26_13 56 42](https://user-images.githubusercontent.com/34873036/43282710-7a63ebf2-90dc-11e8-8e91-2e7067af0d37.png)
![2018-07-26_13 56 59](https://user-images.githubusercontent.com/34873036/43282723-88ca83f4-90dc-11e8-9c8a-75e8d2de507b.png)
Oxygen gear doesn't render on player model

![2018-07-26_13 59 06](https://user-images.githubusercontent.com/34873036/43282749-9ae18b5a-90dc-11e8-8bce-77d197068482.png)
![2018-07-26_13 59 10](https://user-images.githubusercontent.com/34873036/43282752-9d35b44e-90dc-11e8-9dd9-e771e160cad5.png)
while falling back down to earth, parachute doesn't render

**Comment by TheQuadShot** [comment]
What happens if you wear just the GC armour and Oxygen gear? I believe that the 1.7.10 code is no longer being updated unless there is something that is game breaking, so if it is an issue it probably won't get fixed, but the final call on that is made by the Dev team.

**Comment by Dynamic225** [comment]
The oxygen gear doesnt render without armor either. I guess for right now I will just try removing one mod at a time to see which one is causing the problem

**Comment by radfast** [comment]
>    I guess for right now I will just try removing one mod at a time to see which one is causing the problem

Yes please do that.  When you know which it is, please let us know the **exact version** of that other mod so that me or other devs can try the same.

The ones you listed already are, indeed, the most likely ones to try.

**Comment by Dynamic225** [comment]
Optifine was not causing the stuff to not render, but it did stop the sky box from changing while blasting off. Trying More Player Models next...

**Comment by Dynamic225** [comment]
Okay, it appears that More Player Models was the problem. Version of More Player Models is 1.7.10b (latest for 1.7.10). The Optifine version is HD U D6 (I was using an older version to be compatible with fastcraft). Problem with sky box persists on latest version of Optine, too (version HD U E7)
